<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QtAddons::QtLogViewer" name="LogView" native="true">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>9</y>
      <width>381</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="TestButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Test message</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PlotButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>380</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot</string>
    </property>
   </widget>
   <widget class="QtAddons::Plotter" name="CurvePlotter" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>191</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QtAddons::ImageViewerWidget" name="ImageView" native="true">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>220</y>
      <width>271</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ShowImageButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>430</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Show image</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>380</y>
      <width>332</width>
      <height>37</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="GetROIButton">
       <property name="text">
        <string>ROI</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="roi_x0"/>
     </item>
     <item>
      <widget class="QSpinBox" name="roi_y0"/>
     </item>
     <item>
      <widget class="QSpinBox" name="roi_x1"/>
     </item>
     <item>
      <widget class="QSpinBox" name="roi_y1"/>
     </item>
    </layout>
   </widget>
   <widget class="ModuleChainConfiguratorWidget" name="ModuleConf" native="true">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>391</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="myListWidget">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>240</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>894</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QtAddons::QtLogViewer</class>
   <extends>QWidget</extends>
   <header location="global">qtlogviewer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QtAddons::Plotter</class>
   <extends>QWidget</extends>
   <header location="global">plotter.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QtAddons::ImageViewerWidget</class>
   <extends>QWidget</extends>
   <header location="global">imageviewerwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ModuleChainConfiguratorWidget</class>
   <extends>QWidget</extends>
   <header location="global">modulechainconfiguratorwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
